### ISSUE TITLE
bp.plan() gets stuck at SHOW TAGS IN ACCOUNT    

### ISSUE THREAD

**Issue by aleenprd** [Titan-Systems/titan#95]
```
[redacted:ACCOUNTADMIN] > 
        SELECT
            CURRENT_ACCOUNT_NAME() as account,
            CURRENT_ACCOUNT() as account_locator,
            CURRENT_USER() as user,
            CURRENT_ROLE() as role,
            CURRENT_AVAILABLE_ROLES() as available_roles,
            CURRENT_SECONDARY_ROLES() as secondary_roles,
            CURRENT_DATABASE() as database,
            CURRENT_SCHEMAS() as schemas,
            CURRENT_WAREHOUSE() as warehouse,
            CURRENT_VERSION() as version
            (1 rows, 0.17s)
[redacted:ACCOUNTADMIN] > SHOW TAGS IN ACCOUNT    (2 rows, 14.75s)
```

I haven't tried the package in a while now. It seems like at least in version 0.8.9 I can't make the plan. It just gets stuck after showing the tags...

**Comment by aleenprd** [comment]
Update. It's not actually stuck-stuck but the performance is terrible. It's just extremely, extremely slow, compared to a month ago or so. It takes ages to complete the plan.

**Comment by teej** [comment]
I'll try to repro.

**Comment by aleenprd** [comment]
> I'll try to repro.

See comment. It actually does not get completely stuck but freezes at that step for a long time. There is perhaps some other task running in the background but it's not logged.

**Comment by aleenprd** [comment]
> I'll try to repro.

Hi teej, I was mistaken, it's just the next statement it is running that is very slow. It shows grants on a role which returns 37k rows, takes 243s, sounds about right.

**Comment by teej** [comment]
Fixed in [v0.10](https://github.com/Titan-Systems/titan/releases/tag/v0.10.0)
